src/main/java/seedu/address/logic/commands/ViewCommand.java

```diff
@@ -0,0 +1,72 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+import seedu.address.ui.ContactDisplay;
+import seedu.address.ui.TestContactDisplay;
+
+/**
+ * Diplays the details of an existing person in the address book.
+ */
+public class ViewCommand extends Command {
+
+    public static final String COMMAND_WORD = "view";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Displays the details of the contact identified "
+            + "by the index number used in the displayed contact list. "
+            + "Example: " + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_VIEW_PERSON_SUCCESS = "Displayed Person: %1$s";
+
+    private final Index index;
+    private final ContactDisplay contactDisplay;
+    private final TestContactDisplay testContactDisplay;
+
+    /**
+     * @param index of the person in the filtered person list to edit
+     * @param contactDisplay to display the person
+     */
+    public ViewCommand(Index index, ContactDisplay contactDisplay) {
+        requireNonNull(index);
+        this.index = index;
+        this.contactDisplay = contactDisplay;
+        this.testContactDisplay = new TestContactDisplay();
+    }
```

Comment:
Consider adding a test to ensure the ViewCommand instance would be created successfully.